src/main/java/at/hannibal2/skyhanni/features/inventory/bazaar/BazaarCancelledBuyOrderClipboard.kt

```diff
@@ -49,12 +49,12 @@ class BazaarCancelledBuyOrderClipboard {
     fun onChat(event: LorenzChatEvent) {
         if (!isEnabled()) return
         val coins = cancelledMessagePattern.matchMatcher(event.message) {
-            group("coins")
+            group("coins").formatInt().addSeparators()
         } ?: return
 
         val latestAmount = latestAmount ?: error("latest amount is null")
         event.blockedReason = "bazaar cancelled buy order clipboard"
-        ChatUtils.chat("Bazaar buy order cancelled. ${latestAmount.addSeparators()} saved to clipboard. ($coins coins)")
+        ChatUtils.chat(" 1111 11 Bazaar buy order cancelled. ${latestAmount.addSeparators()} saved to clipboard. ($coins coins)")
```

Comment:
```suggestion
        ChatUtils.chat("Bazaar buy order cancelled. ${latestAmount.addSeparators()} saved to clipboard. ($coins coins)")
```

any reason for this

Reply:
So that I can see that the hot swap worked 🤦